ST662_Group6.ipynb

```diff
@@ -2,40 +2,600 @@
+    "df_Group6.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array(['OH', 'TN', 'FL', 'PA', 'CA', 'TX', 'VA', 'MO', 'GA', 'NY', 'CO',\n",
+       "       'WA', 'MN', 'AL', 'IL', 'NC', 'OK', 'MD', 'MI', 'LA', 'SC', nan,\n",
+       "       'OR', 'NH', 'IN', 'NJ', 'DC', 'UT', 'KY', 'NV', 'MA', 'AA', 'RI',\n",
+       "       'KS', 'DE', 'WI', 'MS', 'NM', 'PR', 'NE', 'CT', 'AR', 'AZ', 'IA',\n",
+       "       'ME', 'HI', 'WV', 'ID', 'ND', 'MT',\n",
+       "       'UNITED STATES MINOR OUTLYING ISLANDS', 'AE', 'WY', 'GU', 'SD',\n",
+       "       'AK', 'AP', 'VI', 'VT', 'MP', 'AS', 'MH'], dtype=object)"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Unique values of States Column\n",
+    "\n",
+    "states_list = df_Group6['State']\n",
+    "states_list.unique()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Diccionary of State initials and Full Name\n",
+    "\n",
+    "state_names = {\n",
+    "    'OH': 'Ohio',\n",
+    "    'TN': 'Tennessee',\n",
+    "    'FL': 'Florida',\n",
+    "    'PA': 'Pennsylvania',\n",
+    "    'CA': 'California',\n",
+    "    'TX': 'Texas',\n",
+    "    'VA': 'Virginia',\n",
+    "    'MO': 'Missouri',\n",
+    "    'GA': 'Georgia',\n",
+    "    'NY': 'New York',\n",
+    "    'CO': 'Colorado',\n",
+    "    'WA': 'Washington',\n",
+    "    'MN': 'Minnesota',\n",
+    "    'AL': 'Alabama',\n",
+    "    'IL': 'Illinois',\n",
+    "    'NC': 'North Carolina',\n",
+    "    'OK': 'Oklahoma',\n",
+    "    'MD': 'Maryland',\n",
+    "    'MI': 'Michigan',\n",
+    "    'LA': 'Louisiana',\n",
+    "    'SC': 'South Carolina',\n",
+    "    'nan': 'Unknown',\n",
+    "    'OR': 'Oregon',\n",
+    "    'NH': 'New Hampshire',\n",
+    "    'IN': 'Indiana',\n",
+    "    'NJ': 'New Jersey',\n",
+    "    'DC': 'Washington DC',\n",
+    "    'UT': 'Utah',\n",
+    "    'KY': 'Kentucky',\n",
+    "    'NV': 'Nevada',\n",
+    "    'MA': 'Massachusetts',\n",
+    "    'AA': 'Armed Forces America',\n",
+    "    'RI': 'Rhode Island',\n",
+    "    'KS': 'Kansas',\n",
+    "    'DE': 'Delaware',\n",
+    "    'WI': 'Wisconsin',\n",
+    "    'MS': 'Mississippi',\n",
+    "    'NM': 'New Mexico',\n",
+    "    'PR': 'Puerto Rico',\n",
+    "    'NE': 'Nebraska',\n",
+    "    'CT': 'Connecticut',\n",
+    "    'AR': 'Arkansas',\n",
+    "    'AZ': 'Arizona',\n",
+    "    'IA': 'Iowa',\n",
+    "    'ME': 'Maine',\n",
+    "    'HI': 'Hawaii',\n",
+    "    'WV': 'West Virginia',\n",
+    "    'ID': 'Idaho',\n",
+    "    'ND': 'North Dakota',\n",
+    "    'MT': 'Montana',\n",
+    "    'UNITED STATES MINOR OUTLYING ISLANDS':'UNITED STATES MINOR OUTLYING ISLANDS',\n",
```

Comment:
Looks fine as long as you're not using variables in Tableau with empty spaces - see https://community.tableau.com/s/question/0D54T00000VAUXXSA5/keys-with-character-space